pkg/apis/crds/karpenter.k8s.aws_ec2nodeclasses.yaml

```
@@ -228,6 +276,9 @@ spec:
                   collection and drift handling is implemented for the old instance
                   profiles on an update.
                 type: string
+                x-kubernetes-validations:
+                - message: Immutable field changed
```

```suggestion
                - message: immutable field changed
```

The rest of the errors have been lowercase